relayer/src/chain/client.rs

```diff
@@ -0,0 +1,28 @@
+//! Data structures and logic to set up IBC client's parameters.
+
+use crate::chain::cosmos;
+use crate::config::ChainConfig;
+
+/// Client parameters for the `build_create_client` operation.
+///
+/// The parameters are specialized for each supported chain type.
+#[derive(Clone, Debug)]
+pub enum ClientSettings {
+    Cosmos(cosmos::client::Settings),
```

Comment:
Neat!

Reply:
I'm not sure if `Cosmos` or `Tendermint` is the right name to use here.
`AnyClientState` uses `Tendermint`, maybe we should use it here too for consistency?

Reply:
This is a very good point. `Tendermint` is more appropriate than `Cosmos`.

Reply:
OTOH the module hierarchy under `chain` uses `cosmos`. This is a relayer API anyway, so we don't have to be super-diligent.